packages/spritesheet/src/SpritesheetLoader.ts

```diff
@@ -53,6 +53,39 @@ export class SpritesheetLoader
             return;
         }
 
+        // Check and add the multi atlas
+        // Heavily influenced and based on https://github.com/rocket-ua/pixi-tps-loader/blob/master/src/ResourceLoader.js
+        // eslint-disable-next-line camelcase
+        if (resource.data?.meta?.related_multi_packs && Array.isArray(resource.data?.meta?.related_multi_packs))
+        {
+            for (const item of resource.data.meta.related_multi_packs)
+            {
+                if (typeof item !== 'string')
+                {
+                    continue;
+                }
+
+                const itemName = item.replace('.json', '');
+                const itemUrl = url.resolve(resource.url.replace(loader.baseUrl, ''), item);
+
+                // Check if the file wasn't already added as multipacks are redundant
+                if (loader.resources[itemName]
+                    || Object.values(loader.resources).some((r) => url.format(url.parse(r.url)) === itemUrl))
+                {
+                    continue;
+                }
+
+                const options: any = {
```

Comment:
Can you do this bit without using `any`? Is there a typing issue you're avoiding or issue you're running into?

Reply:
removed the `any` type and found that I had actually a bug on `loadType`

---
Since we are here, this is the code for my loadOptions
```ts
const options = {
    crossOrigin: resource.crossOrigin,
    loadType: LoaderResource.LOAD_TYPE.XHR,
    xhrType: LoaderResource.XHR_RESPONSE_TYPE.JSON,
    parentResource: resource,
};
```

and this is the code for the regular image file for the spritesheet loadOptions
```ts
const loadOptions = {
    crossOrigin: resource.crossOrigin,
    metadata: resource.metadata.imageMetadata,
    parentResource: resource,
};
```

Should I remove the `loadType` and `xhrType` (since it should be automagically guessed) and pass through the metadata field?

Reply:
Yes, for JSON you don't need to set the load type.

Reply:
removed them then 👍